templates/trader.yaml

```diff
@@ -2,17 +2,17 @@ name: "Trader Agent"
 description: "A single-agent service (sovereign agent) placing bets on Omen"
 hash: bafybeidgjgjj5ul6xkubicbemppufgsbx5sr5rwhtrwttk2oivp5bkdnce
 image: https://operate.olas.network/_next/image?url=%2Fimages%2Fprediction-agent.png&w=3840&q=75
-trader_version: v0.18.1
-configuration:
-  nft: bafybeig64atqaladigoc3ds4arltdu63wkdrk3gesjfvnfdmz35amv7faq
-  rpc: http://localhost:8545 # User provided
-  agent_id: 14
-  threshold: 1 # TODO: Move to service component
-  use_staking: false # User provided
-  cost_of_bond: 10000000000000000
-  olas_cost_of_bond: 10000000000000000000
-  olas_required_to_stake: 10000000000000000000
-  monthly_gas_estimate: 10000000000000000000
-  fund_requirements:
-    agent: 100000000000000000
-    safe: 5000000000000000000
+service_version: v0.18.1
+home_chain_id: 100
+configurations:
+  100:
+    staking_program_id: pearl_beta
+    nft: bafybeig64atqaladigoc3ds4arltdu63wkdrk3gesjfvnfdmz35amv7faq
+    rpc: http://localhost:8545 # User provided
+    threshold: 1 # TODO: Move to service component
+    use_staking: false # User provided
+    cost_of_bond: 10000000000000000
```

Comment:
Same comment here for migration to Pearl Beta - seems the parameters are not correct.